src/ralph/api/auth/oidc.py

```diff
@@ -98,7 +98,7 @@ def get_authenticated_user(
     """Decode and validate OpenId Connect ID token against issuer in config.
 
     Args:
-        token (str): Base64 encoded OIDC Token. This is invoked behind the
+        auth_header (str): Base64 encoded OIDC Token. This is invoked behind the
```

Comment:
I am not sure the description is up-to-date.

Reply:
Oh indeed, missed it in my review!
```suggestion
        auth_header (str): Authentication header containing the Base64 encoded OIDC Token. This is invoked behind the
```